graph/multigraph.go

```diff
@@ -19,78 +19,83 @@ type WeightedLine interface {
 
 // Multigraph is a generalized multigraph.
 type Multigraph interface {
-	// Has returns whether the node exists within the multigraph.
-	Has(Node) bool
+	// Has returns whether the node with the given ID exists
+	// within the multigraph.
+	Has(id int64) bool
 
 	// Nodes returns all the nodes in the multigraph.
 	Nodes() []Node
 
 	// From returns all nodes that can be reached directly
-	// from the given node.
-	From(Node) []Node
+	// from the node with the given ID
+	From(id int64) []Node
 
 	// HasEdgeBetween returns whether an edge exists between
-	// nodes x and y without considering direction.
-	HasEdgeBetween(x, y Node) bool
+	// nodes with IDs xid and yid without considering direction.
+	HasEdgeBetween(xid, yid int64) bool
 
-	// Lines returns the lines from u to v if any such lines
-	// exist and nil otherwise. The node v must be directly
-	// reachable from u as defined by the From method.
-	Lines(u, v Node) []Line
+	// Lines returns the edge from u to v, with IDs uid and vid,
+	// if such an edge exists and nil otherwise. The node v
```

Comment:
Was it intentional to change from line to edge in the description?

Reply:
No, it wasn't. Fixed, thanks.gr